sox.features.js

```diff
@@ -1254,11 +1254,22 @@
         });
       }
 
-      addLabels();
+      function addQuestionStateInReview() {
+        sox.warn("called")
+        const $anchor = $('.summary h2 a');
+        const $aText = $anchor.html();
+        sox.warn($aText)
```

Comment:
and this! we could keep it if it seems useful for debugging in future but maybe we could change to `sox.debug` and add some text like `addQuestionStateInReview anchor text:` to the beginning?